app/models/favorite.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'user'
+require_relative 'tweet'
+
+class Favorite
+  attr_reader :id, :user_id, :tweet_id
+  def initialize(id:, user_id:, tweet_id:)
+    @id = id
+    @user_id = user_id
+    @tweet_id = tweet_id
+  end
+
+  def present?
+    @id != nil
+  end
+
+  def self.build_by_user_and_tweet(user:, tweet:, id:)
```

Comment:
IMO/Idiomatic

スタンプ結合になっているので、以下のようにしようか。
```suggestion
  def self.build_by_user_and_tweet(user_id:, tweet_id:, id:)
```

Ref: https://qiita.com/dekamintv/items/ebe9c0f33a2851e6aa9f#%E3%82%B9%E3%82%BF%E3%83%B3%E3%83%97%E7%B5%90%E5%90%88